advanced_source/usb_semisup_learn.py

```diff
@@ -17,7 +17,7 @@
 This tutorial will walk you through the basics of using the USB lighting
 package.
 Let's get started by training a ``FreeMatch``/``SoftMatch`` model on
-CIFAR-10 using pretrained ViT!
+CIFAR-10 using pretrained Vision Transformers (ViT)!
```

Comment:
I wrote out what SSL and ViT stood for (please double check) since they are used throughout the tutorial and initially I didn't know what they meant. Maybe for ML experts they are obvious...